client/syntaxes/coq.tmLanguage

```diff
@@ -125,7 +125,7 @@
 
         <dict>
             <key>match</key>
-            <string>\b(Hint|Constructors|Resolve|Rewrite|Proof|Ltac|Qed|Defined|Implicit|Set|Arguments|Tactic Notation|Notation|Section|Module Type|Module|End|Variables|Record|Check|Print|Eval)\b</string>
+            <string>(Hint|Constructors|Resolve|Rewrite|Proof|Ltac|Qed|Defined|Implicit|Set|Tactic Notation|Notation|Reserved Notation|Module Type|Module|End|Variables|Check|Print|Eval|Next Obligation)</string>
```

Comment:
I see you change the list here too, but again, I'm not sure what's the idea behind this list. They are not keywords AFAIK, or at least not all of them.